verify_rpms/cmd_runner.py

```diff
@@ -0,0 +1,23 @@
+from subprocess import run, CompletedProcess, CalledProcessError
+from typing import List
```

Comment:
As far as I understand it, with the current python version we're using, we can drop this import and replace `List` with `list`.

Reply:
Done